Add about me example page

ColeTownsend opened the pull request:
This is how you would create a page at the route `/about/`

To link to this you would include it in your site.data.navigation just as you have Art and Math.

gkehne commented:
Cole,

Thanks so much for your help! I had been bumbling through various people's
guides, reading (seemingly) contradictory stuff, and unsure about how to
make sure changes that I implemented and viewed locally would show up on
the website itself.

Your demo file appeared locally when I navigated to
http://localhost:4000/about/, but for some reason wouldn't show up when I
pushed it and went to gregorykehne.com/about/ . But I added "permalink:
/about/" to the headspace, and now it seems to be working!? This is a huge
relief. Now time to make everything pretty...

On Tue, Aug 16, 2016 at 9:59 AM, Cole Townsend <notifications@github.com>
wrote:

> This is how you would create a page at the route /about/
>
> To link to this you would include it in your site.data.navigation just as
> you have Art and Math.
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/gkehne/gkehne.github.io/pull/1
> Commit Summary
>
>    - Add about me example page
>
> File Changes
>
>    - *A* _pages/about/index.md
>    <https://github.com/gkehne/gkehne.github.io/pull/1/files#diff-0> (6)
>
> Patch Links:
>
>    - https://github.com/gkehne/gkehne.github.io/pull/1.patch
>    - https://github.com/gkehne/gkehne.github.io/pull/1.diff
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gkehne/gkehne.github.io/pull/1>, or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APsVd7lTHUuTFlA1c07FbJ_WRzzdKdrpks5qgcIngaJpZM4JlbMk>
> .
>

ColeTownsend commented:
Ah so sometimes you need to use a site.url variable to append that for when
you have it lige
On Sun, Aug 21, 2016 at 2:19 PM Gregory Kehne <notifications@github.com>
wrote:

> Cole,
>
> Thanks so much for your help! I had been bumbling through various people's
> guides, reading (seemingly) contradictory stuff, and unsure about how to
> make sure changes that I implemented and viewed locally would show up on
> the website itself.
>
> Your demo file appeared locally when I navigated to
> http://localhost:4000/about/, but for some reason wouldn't show up when I
> pushed it and went to gregorykehne.com/about/ . But I added "permalink:
> /about/" to the headspace, and now it seems to be working!? This is a huge
> relief. Now time to make everything pretty...
>
> On Tue, Aug 16, 2016 at 9:59 AM, Cole Townsend <notifications@github.com>
> wrote:
>
> > This is how you would create a page at the route /about/
> >
> > To link to this you would include it in your site.data.navigation just as
> > you have Art and Math.
> > ------------------------------
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/gkehne/gkehne.github.io/pull/1
> > Commit Summary
> >
> > - Add about me example page
> >
> > File Changes
> >
> > - *A* _pages/about/index.md
> > <https://github.com/gkehne/gkehne.github.io/pull/1/files#diff-0> (6)
> >
> > Patch Links:
> >
> > - https://github.com/gkehne/gkehne.github.io/pull/1.patch
> > - https://github.com/gkehne/gkehne.github.io/pull/1.diff
> >
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub
> > <https://github.com/gkehne/gkehne.github.io/pull/1>, or mute the thread
> > <
> https://github.com/notifications/unsubscribe-auth/APsVd7lTHUuTFlA1c07FbJ_WRzzdKdrpks5qgcIngaJpZM4JlbMk
> >
> > .
> >
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gkehne/gkehne.github.io/pull/1#issuecomment-241272822>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA8SBuYU1Zhlr-Khx3SF1F5Q7LTd4YCkks5qiJa6gaJpZM4JlbMk>
> .
>
